src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1030,6 +1175,9 @@ namespace
 
         // Retrieve the light's parameters.
         GenLight* light_object = dynamic_cast<GenLight*>(object_state.obj);
+        if (light_object == nullptr)
```

Comment:
When will this happen? What happened before?

Reply:
This fixes #197. Corona lights return nullptr.

Reply:
Oh ok, cool.